www/src/components/TwitterWall/tweets.ts

```diff
@@ -0,0 +1,98 @@
+export const tweets = [
+  [
+    {
+      id: 1,
```

Comment:
What is the `id` for here? If it's to provide a unique `key`, `handle` should be unique unless we have multiple tweets from the same author (which seems odd to have on a tweet wall)

Reply:
I think this needs to be the actual id of the tweet so we can link it